.github/workflows/source-and-docs-release.yml

```diff
@@ -18,27 +36,36 @@ on:
 
 name: "Build Python source and docs artifacts"
 
+# Set from inputs for workflow_dispatch, or set defaults to test push/PR events
+env:
+  GIT_REMOTE: ${{ github.event.inputs.git_remote || 'python' }}
+  GIT_COMMIT: ${{ github.event.inputs.git_commit || '2268289a47c6e3c9a220b53697f9480ec390466f' }}
+  CPYTHON_RELEASE: ${{ github.event.inputs.cpython_release || '3.13.0b1' }}
+
 jobs:
   verify-input:
     runs-on: ubuntu-22.04
+    outputs:
+      # Needed because env vars are not available in the build-docs check below
+      cpython_release: ${{ env.CPYTHON_RELEASE }}
     steps:
       - name: "Workflow run information"
         run: |
-          echo "git_remote: ${{ inputs.git_remote }}"
-          echo "git_commit: ${{ inputs.git_commit }}"
-          echo "cpython_release: ${{ inputs.cpython_release }}"
+          echo "git_remote: $GIT_REMOTE"
+          echo "git_commit: $GIT_COMMIT"
+          echo "cpython_release: $CPYTHON_RELEASE"
 
-      - name: "Checkout ${{ inputs.git_remote }}/cpython"
+      - name: "Checkout ${{ env.GIT_REMOTE }}/cpython"
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
-          repository: "${{ inputs.git_remote }}/cpython"
-          ref: "v${{ inputs.cpython_release }}"
+          repository: "${{ env.GIT_REMOTE }}/cpython"
+          ref: "v${{ env.CPYTHON_RELEASE }}"
           path: "cpython"
 
       - name: "Verify CPython commit matches tag"
         run: |
-          if [[ "${{ inputs.git_commit }}" != "$(cd cpython && git rev-parse HEAD)" ]]; then
-            echo "expected git commit ('${{ inputs.git_commit }}') didn't match tagged commit ('$(git rev-parse HEAD)')"
+          if [[ "${{ env.GIT_COMMIT }}" != "$(cd cpython && git rev-parse HEAD)" ]]; then
```

Comment:
Since we're using environment variables now instead of GitHub Action expressions, can we replace the "${{ NAME }}" with "$NAME" when we're running commands?